src/halmos/utils.py

```diff
@@ -314,7 +317,7 @@ def hexify(x):
     if isinstance(x, str):
         return re.sub(r"\b(\d+)\b", lambda match: hex(int(match.group(1))), x)
     elif isinstance(x, int):
-        return hex(x)
+        return f"0x{x:02x}"
```

Comment:
just curious, is this only about formatting, or also about (micro-)performance?

Reply:
just formatting